src/util/util.go

```diff
@@ -58,7 +58,8 @@ func GetFullCommandOutput(cmd ...string) (string, error) {
 	return strings.TrimSpace(string(output)), err
 }
 
-var openBrowserCommands = []string{"xdg-open", "open"}
+var openBrowserCommands = []string{"xdg-open", "open", "cygstart",
+	"x-www-browser", "firefox", "opera", "mozilla", "netscape"}
 var missingOpenBrowserCommandMessages = []string{
 	"Opening a browser requires 'open' on Mac or 'xdg-open' on Linux.",
```

Comment:
I don't want to mention all the different commands anymore, since there are so many now. Please let me know what you think about the new error message.